core/sail/lucene/src/main/java/org/eclipse/rdf4j/sail/lucene/LuceneDocument.java

```diff
@@ -8,15 +8,29 @@
 package org.eclipse.rdf4j.sail.lucene;
 
 import java.text.ParseException;
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
A minor thing, but: our [code conventions](https://github.com/eclipse/rdf4j/blob/master/.github/CONTRIBUTING.md#code-formatting) don't use wildcard imports, could you roll this back here (and in other places where this has been changed)?